deepchem/utils/electron_sampler.py

```diff
@@ -191,6 +194,7 @@ def move(self,
         float
             accepted move ratio of the MCMC steps.
         """
+        self.sampled_electrons = np.array([])

```

Comment:
We should document this in the docstring as an attribute since we are testing it in the unit tests